storage/tail/28-tail.js

```diff
@@ -37,6 +37,7 @@ module.exports = function(RED) {
                 });
 
                 node.tail.on("error", function(err) {
+                    node.status({ fill: "red",shape:"ring", text: "error" });
```

Comment:
I think we may need to make all strings i18n translated

Reply:
Fixed with 4256a76 and ac40455